src/components/CategoryCard.jsx

```diff
@@ -0,0 +1,24 @@
+import { Link } from "react-router-dom";
+
+const CategoryCard = ({ categories }) => {
+  return (
+    <div>
+      <ul className="category">
+        {categories.map((category) => (
+          <li key={category.slug} className="individualCategory">
+            <h3>{category.slug}</h3>
+
+            <p>
+              <strong>Description:</strong> {category.description}
+            </p>
+            <Link to={`/categories/${category.slug}`}>
+              <button>View Games</button>
```

Comment:
No need to put button's inside of Links as this is misleading. The Link is clickable already and the button is redundant so you can take it out. 

You can put a class on the Link to style it however you'd like FYI